ember-intl/addon/helpers/format-date.ts

```diff
@@ -1,18 +1,21 @@
-import { getOwner } from '@ember/application';
 import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
```

Comment:
I didn't write `import { service } from '@ember/service';`, because `ember-intl` still supports `ember-source@3.28`.

https://blog.emberjs.com/ember-4-1-released/